src/main/java/net/minecraftforge/client/ForgeHooksClient.java

```diff
@@ -172,9 +172,9 @@ public static boolean onDrawBlockHighlight(WorldRenderer context, ActiveRenderIn
         return MinecraftForge.EVENT_BUS.post(new DrawHighlightEvent(context, info, target, partialTicks, matrix, buffers));
     }
 
-    public static void dispatchRenderLast(WorldRenderer context, MatrixStack mat, float partialTicks)
+    public static void dispatchRenderLast(WorldRenderer context, MatrixStack mat, float partialTicks, Matrix4f projectionMatrix, long finishTimeNano)
```

Comment:
This is a binary breaking change, please keep the old method and mark it for removal in 1.16